classes/services/queryBuilders/UserListQueryBuilder.inc.php

```diff
@@ -364,7 +364,7 @@ public function get() {
 		if (!empty($this->getReviewerData)) {
 			$q->leftJoin('review_assignments as ra', 'u.user_id', '=', 'ra.reviewer_id');
 			$this->columns[] = Capsule::raw('MAX(ra.date_assigned) as last_assigned');
-			$this->columns[] = Capsule::raw('(SELECT SUM(CASE WHEN ra.date_completed IS NULL THEN 1 ELSE 0 END) FROM review_assignments AS ra WHERE u.user_id = ra.reviewer_id) as incomplete_count');
+			$this->columns[] = Capsule::raw('(SELECT SUM(CASE WHEN ra.date_completed IS NULL AND ra.declined<>1 THEN 1 ELSE 0 END) FROM review_assignments AS ra WHERE u.user_id = ra.reviewer_id) as incomplete_count');
```

Comment:
Looks great. Should any of the other compiled values here be adjusted as well?

I'm not sure if it is possible for `complete_count` to include declined reviews, but if so, they should probably be excluded here too.

Likewise with `average_time`. We should probably exclude declined reviews from that since `date_completed` is never set?

Reply:
May I suggest to also exclude declined review assignments from active reviews count? Line 398 would look like this:

$subqueryStatement = '(SELECT SUM(CASE WHEN ra.date_completed IS NULL  AND ra.declined <> 1 THEN 1 ELSE 0 END) FROM review_assignments AS ra WHERE u.user_id = ra.reviewer_id)';

Thanks!

Reply:
Likewise with `reviewsActive`, they should exclude declined reviews.

Reply:
Oh, good spot @Generalelektrix. The code changed in this PR changes the counts that are returned. But the code directly below represents the query parameters that can be passed, like "if has >X active reviews", so that should be updated too.